lib/models/Alien.js

```diff
@@ -0,0 +1,94 @@
+const pool = require('../utils/pool');
+
+module.exports = class Alien {
+  id;
+  number_of_eyes;
+  color;
+  location_sighted;
+
+  constructor(row) {
+    this.id = row.id;
+    this.number_of_eyes = row.number_of_eyes;
```

Comment:
We should use the constructor here to change the snake_cased database column names to camelCased JavaScript names. for example `this.numberOfEyes = row.number_of_eyes`